source/module_cell/read_atoms.cpp

```diff
@@ -606,6 +606,9 @@ bool UnitCell::read_atom_positions(std::ifstream &ifpos, std::ofstream &ofs_runn
                                                 }
 												else if ( tmpid == "mag" || tmpid == "magmom")
 												{
+#ifndef __CMD
```

Comment:
What does this flag mean?

Reply:
> What does this flag mean?

This flag is used for UT of module_md. I don't know detail.